bullet_train-super_scaffolding/lib/scaffolding.rb

```diff
@@ -7,6 +7,7 @@ def self.mysql?
 
   def self.valid_attribute_type?(type)
     [
+      "active_storage_image",
```

Comment:
@jagthedrummer I don't think we'll need "active_storage_image" here (or "cloudinary_image" from this array either) since we want to use just `image` as a Super Scaffolding type.

For that reason, we should also register "image" as a partial type in `attribute.rb`:

https://github.com/bullet-train-co/bullet_train-core/blob/9983ae6944b40468ea5a7d460371d1626ae5cfe0/bullet_train-super_scaffolding/lib/scaffolding/attribute.rb#L140-L142

We could change it to this (assuming that we don't want to use `cloudinary_image` for backwards compatability):
```ruby
when "image"
  options[:height] = 200
  "image"
```

Since it's not currently registered there, I get the following when trying to scaffold an ActiveStorage image:

```
> rails g model Project team:references title:string foo:attachment
> bin/super-scaffold crud Project Team title:text_field foo:image
```

```
rake aborted!
Invalid field type: image.
/home/gazayas/work/bt/bullet_train/local/bullet_train-core/bullet_train-super_scaffolding/lib/scaffolding/attribute.rb:166:in `partial_name'
/home/gazayas/work/bt/bullet_train/local/bullet_train-core/bullet_train-super_scaffolding/lib/scaffolding/transformer.rb:656:in `block in add_attributes_to_various_views'
/home/gazayas/work/bt/bullet_train/local/bullet_train-core/bullet_train-super_scaffolding/lib/scaffolding/transformer.rb:632:in `each'
/home/gazayas/work/bt/bullet_train/local/bullet_train-core/bullet_train-super_scaffolding/lib/scaffolding/transformer.rb:632:in `each_with_index'
/home/gazayas/work/bt/bullet_train/local/bullet_train-core/bullet_train-super_scaffolding/lib/scaffolding/transformer.rb:632:in `add_attributes_to_various_views'
/home/gazayas/work/bt/bullet_train/local/bullet_train-core/bullet_train-super_scaffolding/lib/scaffolding/transformer.rb:1409:in `scaffold_crud'
/home/gazayas/work/bt/bullet_train/local/bullet_train-core/bullet_train-super_scaffolding/lib/bullet_train/super_scaffolding/scaffolders/crud_scaffolder.rb:91:in `run'
/home/gazayas/work/bt/bullet_train/local/bullet_train-core/bullet_train-super_scaffolding/lib/scaffolding/script.rb:110:in `<main>'
<internal:/home/gazayas/.rbenv/versions/3.2.2/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:37:in `require'
<internal:/home/gazayas/.rbenv/versions/3.2.2/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:37:in `require'
/home/gazayas/work/bt/bullet_train/local/bullet_train-core/bullet_train-super_scaffolding/lib/bullet_train/super_scaffolding.rb:26:in `run'
/home/gazayas/work/bt/bullet_train/local/bullet_train-core/bullet_train-super_scaffolding/lib/tasks/bullet_train/super_scaffolding_tasks.rake:10:in `block (2 levels) in <main>'
Tasks: TOP => bullet_train:super_scaffolding
(See full trace by running task with --trace)
```

Reply:
I'm not sure if I explained all of that too well, so feel free to ask away if something doesn't make sense.

Reply:
Made perfect sense, @gazayas. Thanks for the assist!

I made the changes you suggested and did a little more clean up. Now scaffolding an image with the (new) documentation is working as expected for both Cloudinary and ActiveStorage.